src/utils/hact/workOrderStatusUpdate/assignOperatives.js

```diff
@@ -15,6 +16,9 @@ export const buildOperativeAssignmentFormData = (
           op.percentage == '-' ? 0 : parseFloat(op.percentage.slice(0, -1)),
       }),
     })),
+    ...(operativesNotes && {
+      comments: `Work order updated - ${operativesNotes}`,
+    }),
```

Comment:
hey, everything looks great, just 1 question: 
don't we have "work order completed/ work order updated" added by b/e? 
Like in "utils/hact/workOrderStatusUpdate/updateWorkOrder" (line 31) we add "variation reason" instead of "Work order updated"